src/main/java/seedu/address/model/Model.java

```diff
@@ -68,7 +68,7 @@ public interface Model {
      * Adds the given member.
```

Comment:
```suggestion
     * Creates the given member.
```
should we change to this?

Reply:
Yes, thanks for spotting it!